torchrl/modules/models/multiagent.py

```diff
@@ -229,6 +229,141 @@ def forward(self, *inputs: Tuple[torch.Tensor]) -> torch.Tensor:
 
         return output
 
+class MultiAgentConvNet(nn.Module):
+    """Multi-agent CNN
+
+    This is a CNN that can operate in Multiagent contexts.
+
+    It expects inputs with shape (*B, n_agents, x, y, channels)
+
+    Args:
+        n_agents (int): number of agents.
+        centralised (bool): If `centralised` is True, each agent will use the inputs of all agents to compute its output
+            (n_agent_inputs * n_agents will be the number of inputs for one agent).
+            Otherwise, each agent will only use its data as input.
+        share_params (bool): If `share_params` is True, the same MLP will be used to make the forward pass
+            for all agents (homogeneous policies). Otherwise, each agent will use a different MLP to process
+            its input (heterogeneous policies).
+        device (str or torch.device, optional): device to create the module on.
+        num_cells (int or Sequence[int], optional): number of cells of every layer in between the input and output. If
+            an integer is provided, every layer will have the same number of cells. If an iterable is provided,
+            the linear layers out_features will match the content of num_cells.
+        kernel_sizes (int, Sequence[Union[int, Sequence[int]]]): Kernel size(s) of the conv network.
+            default: 5
+        strides (int or Sequence[int]): Stride(s) of the conv network. If iterable, the length must match the
+            depth, defined by the num_cells or depth arguments.
+            default: 2
+        activation_class (Type[nn.Module]): activation class to be used.
+            default: nn.ELU
+        **kwargs: for :class:`torchrl.modules.models.ConvNet` can be passed to customize the ConvNet.Args:
+
+    Examples:
+        >>> import torch
```

Comment:
Could we add some examples for sharing and centralisation?
Cause they may not be intuitive features immediately.
somthing similar to the ones in the mlp class

Reply:
Added